index.js

```diff
@@ -152,6 +155,7 @@ AddonManager.addInstallListener({
     state.updates = state.updates.filter(u => u.id !== addon.id);
     button.badge = state.updates.length;
     app.send('addon-install:install-ended', formatInstallData(install, addon));
+    pingServer('experiment.installed', addon);
```

Comment:
It would be nice to use the exact same event names for both the front-end and the server. It would also be nice to hide both `app.send` and `pingServer` behind a single method, something like `trigger` (since we're firing an event, in an abstract sense).

It's totally fine to file a followup issue and deal with this later.